File: algorithm/String/prgs 12951 JadenCase 문자열 만들기.py
#프로그래머스 JadenCase 문자열 만들기

def solution(s):
    s = s.split(' ')
    arr = []
    for i in s:
        new = i.capitalize()
        arr.append(new)
    return ' '.join(arr)

print(solution("3people unFollowed me"))
print(solution("for the last week"))

File: js/Programers/Lv2/prgs 12951 JadenCase 문자열 만들기.js
//프로그래머스 12951 JadenCase 문자열 만들기
function solution(s) {
    let arr = s.split(' ').map((value)=>{
        if (value) {return value[0].toUpperCase() + value.slice(1).toLowerCase()}
    })
    let answer;
    answer = arr.join(' ');
    return answer;
}


/**
공백에 따라서 나눠서 볼건데, 공백이 연속해서 나올 수 있음.
**/
